libs/code-actions/src/lib.rs

```diff
@@ -2,6 +2,7 @@ mod code_actions_provider;
 mod completions;
 mod error;
 mod references;
+mod test_utils;
```

Comment:
this module could be declared only on test runs (check rust macros for that) in order to remove all the "#[allow(dead_code)] everywhere